meetuppizza/tests.py

```diff
@@ -57,68 +60,60 @@ def setUp(self):
               "state": "NY"
             },
           }
-        }
-    self.patcher = patch('meetup.services.meetup_api_lookup_agent.MeetupApiLookupAgent')
+        ]
+    self.patcher = patch('meetuppizza.views.MeetupApi')
     self.mock_agent = self.patcher.start()
     self.mock_agent.return_value.get_response.return_value.json.return_value = self.meetup_info
```

Comment:
Here's the same train we saw before. Could that intermediate abstraction be helpful here, too?